doc/TRUEConnector/prerequisite.md

```diff
@@ -11,25 +11,130 @@ To have secure and certification compliant environment, following prerequisites
+Open the config file /etc/ssh/sshd_config with your favorite editor and add these lines:
+
+```
+PermitRootLogin no
+AllowGroups ssh_login_group
+
+```
+
+### Disable password login
+
+You probably want to disable password login to avoid someone logging in from somewhere else then the hosts you have configured in the authorized_keys file. To disable password authentication globally for all users, the following 3 settings need to be changed to "no" in the /etc/ssh/sshd_config file:
+
+```
+ChallengeResponseAuthentication no
+PasswordAuthentication no
+UsePAM no
+AllowUsers bob alice...
+```
+
+### Create SSH public and private keys for user accessing host machine
+
+As a first step, a key-pair needs to be created. This is usually done on the server. 
+With the following command a new key-pair is created. 
+
+```
+ssh-keygen -t rsa -b 4096 -f ~/.ssh/desktop_key-rsa
+```
+
+In order to create the key, you will be asked for a password. This is the password for your key. It is recommended and considered as best practice (and also security related) to enter passphrase. It will be used as security step, avoiding the usage of a stolen or lost private key. The result of this command should be two files. The file "\~/.ssh/desktop_key-rsa" which is the private-key file, and the file "~/.ssh/desktop_key-rsa.pub" which contains your public-key file. 
+This public-key and private-key will be securely transferred to the client. This means that keys are transferred to the client machine without exposing the content of the file, following best practices for delivering files containing sensitive data, such are password protected zip archive, uploading to some storage, and providing link to the responsible user, admin approaching to the client and copying key file from USB stick, or whatever is applicable and most suitable for the company.
+
+public-key needs to be added to the authorized keys. To make sure we do not override any already configured authorized key, 
+we add the public-key to the authorized_keys file. If the file does not yet exist, it will create it automatically:
+
+```
+cat desktop_key-rsa.pub >>~/.ssh/authorized_keys
+```
+
+### Authenticate with SSH key
+
+The format of the "authorized_keys" file is as follows:
+
+**options keytype base64-encoded-key comment**
+
+options can contain filter like 
+
+```
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABA...etc...mnMo7n1DD username
+```
+
+This will enforce that user can log in into server using key authentication.
+
+Once public-key is uploaded to authorized_keys, user requesting access to the server should securely receive public and private key.
+User can verify that connection with following:
+
+```
+ssh username@server.example.com -v -i .ssh/desktop_key-rsa
+```
+
+Note: username used in command to connect to the server needs to be added in sshd deamon configuration file and passphrase for that key needs to be entered when prompted.
+
+```
+AllowUsers bob alice...
```

Comment:
To change bob and alice, and use some other name, not to make confusion with connector users?